json_schemas/framework-agreement-details.json

```diff
@@ -15,6 +15,57 @@
       "minLength": 1,
       "type": "string"
     },
+    "frameworkRefDate": {
```

Comment:
What does 'Ref' stand for? Can we expand this key name or is it already used somewhere?